generated/python/gapic-google-cloud-language-v1beta1/google/cloud/gapic/language/v1beta1/enums.py

```diff
@@ -0,0 +1,277 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Wrappers for protocol buffer enum types."""
+
+
+class DocumentType(object):
```

Comment:
@geigerj Please note that `DocumentType` and `EntityType` were both auto-generated as `Type`. I manually disambiguated them.

There was a similar issue in Vision (also `Type`) and in Datastore (`Operator`).

We need to create an issue to disambiguate these, but it may not need to be fixed right away if we keep hand-editing around the problem.